llvm/unittests/ExecutionEngine/JITLink/AArch32Tests.cpp

```diff
@@ -242,3 +246,41 @@ TEST(AArch32_Relocations, Thumb_MovtAbs) {
     }
   }
 }
+
+/// Write immediate value to the top halfword of the destination register
+TEST(AArch32_Relocations, Arm_MovtAbs) {
+  static_assert(isUInt<16>(65535), "Max value");
+  static_assert(!isUInt<16>(65536), "First overflow");
+
+  constexpr uint32_t ImmMask = FixupInfo<Arm_MovtAbs>::ImmMask;
+  constexpr uint32_t RegMask = FixupInfo<Arm_MovtAbs>::RegMask;
+
+  static std::array<uint8_t, 3> Registers{0, 5, 12};
+  static std::array<uint32_t, 3> MemPresets{
+      0xfeeffff7, // common
+      0x00000000, // zeros
+      0xffffffff, // ones
+  };
+
+  auto EncodeDecode = [](uint64_t In, MutableWord &Mem) {
```

Comment:
The Windows MSVC builder complains: `'ImmMask' cannot be implicitly captured because no default capture mode has been specified`
But adding the explicit capture Clang warns: `Lambda capture 'ImmMask' is not required to be captured for this use`

Seems to be a known implementation divergence: https://stackoverflow.com/questions/52416362/unused-lambda-capture-warning-when-capture-is-actually-used Interestingly, I did the same in the `Arm_Call_Bare` test and I am not aware the issue ever came up.

Well, I guess we should just specify default value capture `[=]` and it will go away. While you are here, could you fix it in line 190 as well please? Thx